### ISSUE TITLE
built in Pandoc

### ISSUE THREAD

**Issue by Reaper10** [olivierkes/manuskript#909]
Can built in Pandoc into  manuskript?

**Comment by TheJackiMonster** [comment]
What exactly do you mean by this? We're already using Pandoc as optional dependency for exporting/importing different formats.

**Comment by Reaper10** [comment]
make so a user doesn't need download separately. this will help with app versions ie android an ios.

**Comment by TheJackiMonster** [comment]
There are no versions of Manuskript for Android or iOS afaik. Otherwise the Windows version includes Pandoc already and Linux distributions have a proper package management to handle this in a preferred way. So I'm not sure where this seems to be an issue currently.

**Comment by Reaper10** [comment]
it's not built in with out installing it
<img width="378" alt="Capture 2" src="https://user-images.githubusercontent.com/8137561/127415796-26e27753-1886-4a4d-8302-fca0c2e87ef0.PNG">
<img width="479" alt="Capture" src="https://user-images.githubusercontent.com/8137561/127415814-cf1d6051-88d1-4c71-ab71-3be9189d2097.PNG">

**Comment by TheJackiMonster** [comment]
You're right, sorry. I've checked the Windows version on my system as well and Pandoc does not seem to be included despite I have added it into the packaging process. Maybe this also correlates with some other issue that LanguageTool wasn't included in the package as well. So for example it could be that the PyInstaller does not include optional dependencies into the package even if they are installed during the packaging-process.

I will try to figure out if this can be adjusted before releasing the next version. So Pandoc can be included in the Windows version properly.

**Comment by Reaper10** [comment]
still not built in as September 6 2023
